templates/patient/card/billing.html.twig

```diff
@@ -2,9 +2,9 @@
 
 {% block content %}
 <div class="container">
-    <div class="row">
-        <div class="col-4">{{ "Patient Balance Due"|xlt }}</div>
-        <div class="col">{{ patientBalance|money|text }}</div>
+    <div class="row"> <!--ALB Changed this to red and bold -->
+        <div class="col-4 font-weight-bold"><font color='red'>{{ "Patient Balance Due"|xlt }}</font></div>
```

Comment:
Don't add `font-weight-bold` here - it's very overused throughout the UI and I am actively working on removing it. The `<font>` tag is long deprecated and there's no need to make this text red, leave it as it.